CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## [1.50](https://github.com/brave/brave-ios/releases/tag/v1.50)
+
+### Web3
+
+- Added support for opening ipns:// scheme URL from other apps. ([#7227](https://github.com/brave/brave-ios/issues/7227))
+- Added domain resolution support for Unstoppable Domains. ([#7089](https://github.com/brave/brave-ios/issues/7089))
+- Added support for IPFS Navigation in omnibox. ([#7013](https://github.com/brave/brave-ios/issues/7013))
+- Added checksum validation of addresses on the Send screen in Brave Wallet. ([#7136](https://github.com/brave/brave-ios/issues/7136))
+
+### General
+
+- Added Brave News Widgets feature for home screen and lock screen. ([#7168](https://github.com/brave/brave-ios/issues/7168))
+- Added capability to add custom filter lists. ([#6841](https://github.com/brave/brave-ios/issues/6841))
+- Added support for scriptlets using uBlock Origin format. ([#7194](https://github.com/brave/brave-ios/issues/7194))
+- Added “Never Saved” section in Logins & Passwords Page when syncing passwords. ([#6991](https://github.com/brave/brave-ios/issues/6991))
```

Comment:
nit: I think we're using different quotes here? `“Never Saved”` vs `"Never Saved"`. Can barely tell the difference though so not the biggest deal.

Reply:
Nice find, fixed!